control-plane-operator/controllers/hostedcontrolplane/konnectivity/reconcile.go

```diff
@@ -92,16 +96,14 @@ func buildKonnectivityServerContainer(image string) func(c *corev1.Container) {
 			"--server-key",
 			cpath(konnectivityVolumeServerCerts().Name, corev1.TLSPrivateKeyKey),
 			"--server-ca-cert",
-			cpath(konnectivityVolumeServerCA().Name, pki.CASignerCertMapKey),
+			cpath(konnectivityVolumeServerCerts().Name, pki.CASignerCertMapKey),
 			"--mode=grpc",
 			"--server-port",
 			strconv.Itoa(KonnectivityServerLocalPort),
 			"--agent-port",
 			strconv.Itoa(KonnectivityServerPort),
 			"--health-port=8092",
 			"--admin-port=8093",
-			"--agent-namespace=kube-system",
-			"--agent-service-account=konnectivity-agent",
 			"--kubeconfig",
 			cpath(konnectivityVolumeKubeconfig().Name, KubeconfigKey),
 			"--authentication-audience=system:konnectivity-server",
```

Comment:
You'll need to remove --kubeconfig and --authentication-audience as well since they're only used for token-based auth. (You'll get an error about token auth)

Reply:
Yep, I figured that out :smile:  Working now :+1: